Index: autodock-vina-1.1.2/build/linux/debug/Makefile
===================================================================
--- autodock-vina-1.1.2.orig/build/linux/debug/Makefile	2011-05-12 13:03:59.108034987 +0200
+++ autodock-vina-1.1.2/build/linux/debug/Makefile	2011-05-12 13:04:34.736211653 +0200
@@ -1,8 +1,8 @@
-BASE=/usr/local
-BOOST_VERSION=1_41
+BASE=/usr
+BOOST_VERSION=1_46
 BOOST_INCLUDE = $(BASE)/include
-C_PLATFORM=-static -pthread
-GPP=/usr/local/bin/g++
+C_PLATFORM= -pthread
+GPP=g++
 C_OPTIONS= -g
 BOOST_LIB_VERSION=
 
